Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```
@@ -0,0 +1,1481 @@
+        name=hr_title,
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.PullRequestComment',
+        outputs=results,
+        raw_response=results
+    )
+
+
+def pull_request_comment_update_command(client: Client, args: Dict) -> CommandResults:
+    """ Updates a comment in a pull request.
+        Args:
+            client: A Bitbucket client.
+            args: Demisto args.
+        Returns:
+            A CommandResult object with a success message.
+    """
+    repo = args.get('repo')
+    pr_id = int(args.get('pull_request_id', ''))
+    comment_id = args.get('comment_id', '')
+    content = args.get('content')
+    if not repo:
+        repo = client.repository
+    body = {
+        'content': {
+            'raw': content
+        }
+    }
+    response = client.pull_request_comment_update_request(repo, pr_id, body, comment_id)
+    return CommandResults(
+        readable_output='The comment was updated successfully',
+        outputs_prefix='Bitbucket.PullRequestComment',
+        outputs=[response],
+        raw_response=[response])
+
+
+def pull_request_comment_delete_command(client: Client, args: Dict) -> CommandResults:
+    """ Deletes a comment in a pull request.
+        Args:
+            client: A Bitbucket client.
+            args: Demisto args.
+        Returns:
+            A CommandResult object with a success message.
+    """
+    repo = args.get('repo')
+    pr_id = int(args.get('pull_request_id', ''))
+    comment_id = args.get('comment_id', '')
+    if not repo:
+        repo = client.repository
+    response = client.pull_request_comment_delete_request(repo, pr_id, comment_id)
+    if response.status_code == 204:
+        return CommandResults(readable_output='The comment was deleted successfully.')
+    else:
+        raise Exception('The command pull-request-comment-delete failed.')
```

Let's return the error that returns from the API.
This error doesn't give us any information, therefore is not useful